online_meetings/report.py

```diff
@@ -21,6 +21,26 @@
 DEFAULT_SLEEP_TIME = ENV.get('DEFAULT_SLEEP_TIME', 10)
 
 
+def to_seconds(duration: str) -> int:
+    """Returns a value of a string duration in seconds
+       This string is expected in HH:MM:SS or MM:SS or SS
+
+    :param timestr: String in a duration format like HH:MM:SS
+    :type timestr: str
+    :return: Value in seconds
+    :rtype: int
+    """
+    total_seconds = 0
+    for second in duration.split(':'):
+        try:
+            total_seconds = total_seconds * 60 + int(second)
+        # If there is an error just return 0
+        except ValueError:
+            logger.warn(f"{str} is not a valid duration")
+            return 0
```

Comment:
This looks like it will work. I was a little confused at first by the variable names, since the value of `total_seconds` doesn't signify seconds until the end. Not sure what would be better.

Reply:
Yeah, it's just used as an accumulator in the for loop.